react.js

```diff
@@ -0,0 +1,138 @@
+module.exports = {
+    'parserOptions': {
+        'ecmaFeatures': {
+            'jsx': true
+        }
+    },
+    'plugins': [
+        'jsdoc',
```

Comment:
I don't think the jsdoc plugin and rules should be related to react. Ideally they should be in the index.js but the problem is that we will end up with a lot of errors in lib-jitsi-meet and in the non-react code of jitsi-meet. I don't think we have the time to fix them right now.

One option would be to just put a FIXME comment here about the problem. Another one would be to create "jsdoc.js" eslint config file for them.

@AkshitKrNagpal @saghul WDYT?

Reply:
I think we should go for creating `jsdoc.js`.
But I have a question. 
Do we want to enforce `jsdoc` for react projects? If so we can modify `react.js` to extend `jsdoc.js`.

Reply:
We want to enforce jsdoc rules for everything. That's why I said that they should be included in the index.js. The problem is that we are not there yet because we have to prepare our source code for this. So I'd prefer to have them in jsdoc.js for now and when we are ready we will move them to index.js. This way we'll be more flexible because we'll be able to enforce them for new non-react projects too.

Reply:
Cool. 👍